cpp/src/file/read_file.cc

```diff
@@ -123,4 +130,32 @@ int ReadFile::read(int32_t offset, char *buf, int32_t buf_size,
     return ret;
 }
 
-}  // end namespace storage
\ No newline at end of file
+}  // end namespace storage
+
+// https://github.com/aleitner/windows_pread/blob/master/src/pread.c
+// We need to find out which license this is under.
```

Comment:
ok, so we can also change or remove this comment?

Reply:
Will take care of that

Reply:
Done